frontend/server/src/test/java/org/pytorch/serve/ModelServerTest.java

```diff
@@ -1244,4 +1248,65 @@ private void testMetricManager() throws JsonParseException, InterruptedException
             }
         }
     }
+
+    private void testTSValidPort()
+            throws InterruptedException, GeneralSecurityException, InvalidSnapshotException,
+                    IOException {
+        //  test case for verifying port range refer https://github.com/pytorch/serve/issues/291
+        server.stop();
+        System.setProperty("tsConfigFile", "src/test/resources/config_port.properties");
+        FileUtils.deleteQuietly(new File(System.getProperty("LOG_LOCATION"), "config"));
```

Comment:
I am curious to know what this file is, that we are deleting.

Reply:
Due to snapshot feature, the first preference will be given to last available snapshot file instead of the properties file set in the system properties. Thus cleaning up the generated snapshot before restart.

Reply:
Why is snapshot given a preference over the runtime configuration? Whats the order of preference for this?

Reply:
The preference order for properties file is as follows :

 - System environment
 - Command line argument
 - Last snapshot
 - Java System Property.